examples/crypto/SP800-108_counter_KDF/main.c

```diff
@@ -0,0 +1,397 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include <psa/crypto.h>
+
+#define CMAC_KEY_LENGTH  (16)
+#define OTUPUT_LENGTH    (42)
+#define AES_BLOCK_SIZE    PSA_BLOCK_CIPHER_BLOCK_LENGTH(PSA_KEY_TYPE_AES)
+#define L_LENGTH 	(sizeof(uint32_t))
```

Comment:
It's defined as 4 in the specification. The fact that it's `sizeof(uint32_t)` is a coincidence.